core/src/main/java/jenkins/tools/ToolConfigurationCategory.java

```diff
@@ -0,0 +1,21 @@
+package jenkins.tools;
```

Comment:
Tempted to make this `@Restricted(NoExternalUse.class)` for a time to first determine how this should be used by plugins. What kinds of configuration should be moved here?